docs/framework/data/adonet/sql/linq/index.md

```diff
@@ -29,8 +29,9 @@ ms.assetid: 73d13345-eece-471a-af40-4cc7a2f11655
  Provides links to Visual Basic and C# samples.  
   
 ## Related Sections  
- [LINQ (Language-Integrated Query)](https://msdn.microsoft.com/library/a73c4aec-5d15-4e98-b962-1274021ea93d)  
- Provides an overview of [!INCLUDE[vbteclinq](../../../../../../includes/vbteclinq-md.md)] technologies.  
+ [Language-Integrated Query (LINQ) - C#](../../../csharp/programming-guide/concepts/linq/index.md)  
```

Comment:
You have this and several other broken links in the build report. Take a look. I'd also break the links in two where you have the C# and VB version. 

```suggestion
 [Language-Integrated Query (LINQ) - C#](../../../../../csharp/programming-guide/concepts/linq/index.md)\
 Provides overviews of LINQ technologies in C#.
 
```

Reply:
@mairaw My LINQ-C# and LINQ-VB links (going up 3 parent folders instead of 5) work in VS Code preview. I also don't understand how you can go up 5 folder levels, since 3 gets you to a top-level folder in the repo. Does using 5 lead to another repo with the same file structure?

Reply:
I just counted how many subfolders you had in here and took a guess on why these links are broken:
The file is docs/framework/data/adonet/sql/linq/index.md

Going up just 3 levels brings you to the data folder.